<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServiceWidget</class>
 <widget class="QWidget" name="ServiceWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>502</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="ObjectInfoWidget" name="objectInfoWidget" native="true"/>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="parametersTab">
      <attribute name="icon">
       <iconset theme="utilities-system-monitor"/>
      </attribute>
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QCheckBox" name="deployedCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Deployed</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="startedCheckBox">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Started</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>View parameters:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="comboBox">
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Default</string>
            </property>
            <property name="icon">
             <iconset theme="emblem-default-symbolic"/>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Current</string>
            </property>
            <property name="icon">
             <iconset theme="network-server-symbolic"/>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="CustomTreeView" name="parametersView"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="resourcesTab">
      <attribute name="icon">
       <iconset theme="text-x-generic"/>
      </attribute>
      <attribute name="title">
       <string>Resources</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="CustomTreeView" name="resourcesView"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Orientation::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeView</class>
   <extends>QTreeView</extends>
   <header>ui/CustomTreeView.h</header>
  </customwidget>
  <customwidget>
   <class>ObjectInfoWidget</class>
   <extends>QWidget</extends>
   <header>ui/ObjectInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
